internal/service/privateendpointregionalmode/resource_private_endpoint_regional_mode_test.go

```diff
@@ -18,31 +18,31 @@ func TestAccPrivateEndpointRegionalMode_basic(t *testing.T) {
 }
 
 func TestAccPrivateEndpointRegionalMode_conn(t *testing.T) {
-	acc.SkipTestForCI(t) // needs AWS configuration
-
 	var (
-		endpointResourceSuffix = "atlasple"
-		resourceSuffix         = "atlasrm"
-		resourceName           = fmt.Sprintf("mongodbatlas_private_endpoint_regional_mode.%s", resourceSuffix)
-		awsAccessKey           = os.Getenv("AWS_ACCESS_KEY_ID")
-		awsSecretKey           = os.Getenv("AWS_SECRET_ACCESS_KEY")
-		providerName           = "AWS"
-		region                 = os.Getenv("AWS_REGION")
-		projectID              = acc.ProjectIDExecution(t)
-		orgID                  = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		projectName            = acc.RandomProjectName()
-		clusterName            = acc.RandomClusterName()
-		clusterResourceName    = "test"
-		clusterResource        = acc.ConfigClusterGlobal(orgID, projectName, clusterName)
-		clusterDataSource      = modeClusterData(clusterResourceName, resourceSuffix, endpointResourceSuffix)
-		endpointResources      = testConfigUnmanagedAWS(
+		endpointResourceSuffix                 = "atlasple"
+		resourceSuffix                         = "atlasrm"
+		resourceName                           = fmt.Sprintf("mongodbatlas_private_endpoint_regional_mode.%s", resourceSuffix)
+		awsAccessKey                           = os.Getenv("AWS_ACCESS_KEY_ID")
+		awsSecretKey                           = os.Getenv("AWS_SECRET_ACCESS_KEY")
+		providerName                           = "AWS"
+		region                                 = os.Getenv("AWS_REGION_LOWERCASE")
+		privatelinkEndpointServiceResourceName = fmt.Sprintf("mongodbatlas_privatelink_endpoint_service.%s", endpointResourceSuffix)
+		clusterDependsOn                       = fmt.Sprintf("%s, %s", resourceName, privatelinkEndpointServiceResourceName)
+		spec1                                  = acc.ReplicationSpec(&acc.ReplicationSpecRequest{Region: os.Getenv("AWS_REGION_UPPERCASE"), ProviderName: providerName, ZoneName: "Zone 1"})
```

Comment:
Added a new variable here, could have used `AWS_REGION` but I find it very confusing (it is `US_EAST_1`) in the CI. 
When working locally, it is more common to use `us-east-1` as that is what is expected for the AWS TF provider